hack/make-rules/verify.sh

```diff
@@ -69,11 +72,19 @@ function is-quick {
 }
 
 function run-cmd {
+  local filename="${2##*/verify-}"
+  local testname="${filename%%.*}"
+  local output="${KUBE_JUNIT_REPORT_DIR:-/tmp/junit-results}"
+  local tr
+
   if ${SILENT}; then
-    "$@" &> /dev/null
+    juLog -output="${output}" -class="verify" -name="${testname}" "$@" &> /dev/null
+    tr=$?
```

Comment:
If we're running with `set -o errexit` on why does this work?

Reply:
we capture the status when calling `run-cmd`, which is why the script doesn't fully exit on failures here.

I don't think we need to do this `tr=$?` and `return ${tr}` stuff, though - we can keep it as it was before; the correct status should be returned regardless.

Reply:
This is likely the reason: https://stackoverflow.com/a/37191242/

Reply:
@ixdy It [wasn't returning the status](https://github.com/kubernetes/kubernetes/pull/58783#issuecomment-360375698), so these were added.

Reply:
wait, how did this ever work?

Reply:
✨bash✨